site/src/data.ts

```diff
@@ -27,9 +27,21 @@ export interface CurrentAnalytics extends Analytics {
   last_updated?: number;
 }
 
+export interface IntegrationDetails {
+  title: string;
+}
+
+export interface IntegrationData {
+  integration: string;
+  installations: number;
+}
+
 export const fetchData = () =>
   fetch("https://analytics-api.home-assistant.io/v1");
 
+export const fetchIntegrationDetails = () =>
+  fetch("https://www.home-assistant.io/integrations.json", { mode: "no-cors" });
```

Comment:
Isn't that already in `window.integrationsPromise`?

Reply:
It is